python/dgl/graphbolt/item_sampler.py

```diff
@@ -2,16 +2,63 @@
 
 from collections.abc import Mapping
 from functools import partial
-from typing import Iterator, Optional
+from typing import Callable, Iterator, Optional
 
 from torch.utils.data import default_collate
 from torchdata.datapipes.iter import IterableWrapper, IterDataPipe
 
+from ..base import dgl_warning
+
 from ..batch import batch as dgl_batch
 from ..heterograph import DGLGraph
 from .itemset import ItemSet, ItemSetDict
+from .minibatch import MiniBatch
+
+__all__ = ["ItemSampler", "minibatcher_default"]
+
+
+def minibatcher_default(batch, names):
+    """Default minibatcher.
+
+    The default minibatcher maps a list of items to a `MiniBatch` with the
+    same names as the items. The names of items are supposed to be provided
+    and align with the data attributes of `MiniBatch`. If any unknown item name
+    is provided, exception will be raised. If the names of items are not
```

Comment:
Other than raise an exception, can we blindly hold the data in minibatch?

Reply:
yes, we can. But I don't think it's a good behavior to add data attributes into `MiniBatch` though it's quite easy to achieve in python via `setattr()`

Reply:
as discussed offline, let's support such a bit **wild** behaviour